### ISSUE TITLE
Auto-suggest does not work with selectors

### ISSUE THREAD

**Issue by holwech** [matteodem/meteor-easy-search#417]
I could be missing something, but this works perfectly when using the normal input component. It also works without any selectors,  but with a selector no suggestions appear at all.

client

``` javascript
Template.tools.helpers({
    commentsIndex: function() {
        return CommentsIndex;
    }
});
```

``` html
<template name='tools'>
    <div class='row'>
        <div class="autosuggest-component">
            {{> EasySearch.Autosuggest index=commentsIndex}}
        </div>
    </div>
</template>
```

Setting the selector-value

``` javascript
setSearchLanguage = function(lang) {
    CommentsIndex.getComponentMethods()
        .addProps('language', lang);
}
```

server

``` javascript
CommentsIndex = new EasySearch.Index({
    collection: Comments,
    fields: ['comment'],
    engine: new EasySearch.MongoDB({
        selector: function (searchObject, options, aggregation) {
            let selector = this.defaultConfiguration().selector(searchObject, options, aggregation);
            selector.language = options.search.props.language;
            return selector;
        }
    })
});

```

**Comment by matteodem** [comment]
This has to do with the autosuggest component not using the component methods.

**Comment by Tzikas** [comment]
+1 Same issue.  Thank you.

**Comment by matteodem** [comment]
This is fixed in the latest version `2.1.6`.
